.github/workflows/linux_tests.yml

```diff
@@ -21,6 +21,10 @@ jobs:
         build_testing:
           - ON
           - OFF
+        # TODO: remove this once MATLAB dependency is removed and mexFunction is replaced
```

Comment:
Worth breaking this out into another issue, also mentioning that we should turn this option from a compile-time option to a command-line option.